contracts/UFragments.sol

```diff
@@ -72,6 +72,8 @@ contract UFragments is ERC20Detailed, Ownable {
 
     uint256 private constant DECIMALS = 9;
     uint256 private constant MAX_UINT256 = ~uint256(0);
+
+    // TODO: This needs to be updated before launch
```

Comment:
I'm not worried about forgetting this, and I'd rather avoid code churn between now and launch as much as possible.

Reply:
👍